src/OrchardCore.Modules/OrchardCore.Resources/wwwroot/Scripts/popper.js

```diff
@@ -7,7 +7,7 @@ function _typeof(obj) { if (typeof Symbol === "function" && typeof Symbol.iterat
 
 /**!
  * @fileOverview Kickass library to create and place poppers near their reference elements.
- * @version 1.14.7
+ * @version 1.15.0
  * @license
```

Comment:
@agriffard could you run npm install again and rebuild the js?

The version of popper and codemirror is not the same as is in the package.json file 

maybe CodeMirror is ready for an update if there is a new version?

Looking forward to seeing this merged 😄

Reply:
`npm i` always restores popper.js 1.15.0, even from a fresh install, I don't know why.

Reply:
@deanmarcussen if you can do it on your machine and push a commit to the PR, please do it.

Reply:
@agriffard I do it as a separate PR https://github.com/OrchardCMS/OrchardCore/pull/4583 as I don't want to confuse your PR with other changes.

However, I get to try this. It looks great!

I have two minor comments

- Why is `Type` greyer than the other selectors?
- Can the item selector stay with the count when it shrinks down? Maybe it is just inline-block or a wrapper

![content-list](https://user-images.githubusercontent.com/13782679/67156734-89510b80-f31a-11e9-84b6-1b7a0e3cc0b4.PNG)